daft/execution/execution_step.py

```diff
@@ -412,7 +412,6 @@ def _handle_file_write(self, input: MicroPartition) -> MicroPartition:
 class WriteDeltaLake(SingleOutputInstruction):
     base_path: str
     large_dtypes: bool
-    current_version: int
```

Comment:
I would leave this in to enable overwrites at a per partition level like specified here https://docs.databricks.com/en/delta/selective-overwrite.html#dynamic-partition-overwrites